app.css

```diff
@@ -0,0 +1,67 @@
+
+*{
+    min-width: 400px;
+}
+.body{
+    font-family: 'Lato', sans-serif;
+    color: rgba(0, 0, 0, 0.774);
+}
+.main{
+    text-align: center;
+    align-items: center;
+    margin: 22px 7px 22px 7px;
+    background-color: rgb(238,238,238);
+}
+.main_figure{
+    background-color: white;
+    margin: auto;
+    width: 98%;
+}
+.main h1{
+    padding-top: 50px;
+    font-size: 43px;
+}
+.main_img{
+    width: 1024;
+    
+}
+@media only screen and (max-width:600px){
+    .main_img{
+        width: 600px;
+    }
+}
+@media only screen and (max-width:800px) {
+    .main_img{
+        width: 800px;
+    }
+}
+@media only screen and (max-width:400px) {
+    .main_img{
+        width: 400px;
+    }
+}
+.main_figure>figcaption{
+    padding: 20px 0px 20px;
+}
+
+
```

Comment:
- [ ] Please do not add blank lines, spaces, or indentations without a reason.
  [Hint] Kindly delete one blank line